### DISCUSSION TITLE
Crear Buyer Persona

### DISCUSSION THREAD

**Discussion by erykzon** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#65] [Ideas]
Hola compañeros,

Comparto un link donde pueden crear su buyer persona , es muy intuitivo y sencillo .

https://www.hubspot.es/make-my-persona

Espero les funciones , vamos por mas!

**Comment by manuel1488** [comment]
Es buena, pero se siente limitada con opciones limitadas. pero gracias por la aportación la usare

**Comment by SandroBorrallez** [comment]
Muchas gracias por el dato.
